static/less/style.less

```diff
@@ -3,6 +3,11 @@
 @darker-bg-color: #dcdcdc;
 @darker-fg-color: #000;
 @link-color: #3366CC;
+@inverted-bg-color: #101010;
+@inverted-fg-color: #fefefe;
+@inverted-darker-bg-color: #000;
+@inverted-darker-fg-color: #dcdcdc;
+@inverted-link-color: #1ec503;
```

Comment:
I just approved another PR that moved color variables to `colors.less`. Please, rebase and fix it.

Reply:
✅